environment.yml

```
@@ -38,7 +38,7 @@ dependencies:
     - pytorch-lightning>=1.5.1
     - git+https://github.com/pytorch/pytorch_sphinx_theme
     - pyupgrade>=2.4
-    - radiant-mlhub>=0.2.1,<0.5
+    - radiant-mlhub>=0.2.1,<0.6
```

```suggestion
    - radiant-mlhub>=0.2.1
```
We don't usually add upper bounds for conda since this file isn't used in production